packages/freezed_lint/test/ensure_build_test.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:build_verify/build_verify.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('ensure_build', () {
+    expectBuildClean(
```

Comment:
Fixes aren't tested. Although I'll admit I haven't thought too much about how to test them.